src/fmeval/eval_algorithms/classification_accuracy.py

```diff
@@ -88,15 +102,24 @@ class ClassificationAccuracyConfig(EvalAlgorithmConfig):
 
     :param valid_labels: The labels of the classes predicted from the model.
     :param converter_fn: Function to process model output to labels, defaults to simple integer conversion.
+    :param binary_average_strategy: `average` to be passed to sklearn's precision and recall scores.
+        This determines how scores are aggregated in the binary classification settings
+        (see https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_score.html).
+        Options are {'micro', 'macro', 'samples', 'weighted', 'binary'} or None, default='binary'.
+    :param positive_label: The label that is considered to be the positive class when computing precision and recall
+        for binary classification problems and the averaging strategy is `binary`. This parameter has no effect
+         for the multiclass case. Default='1'.
     :param multiclass_average_strategy: `average` to be passed to sklearn's precision and recall scores.
-        This determines how scores are aggregated in the multiclass classification setting
+        This determines how scores are aggregated in multiclass classification settings
         (see https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_score.html).
-        Options are {'micro', 'macro', 'samples', 'weighted', 'binary'} or None, default='micro'.
+        Options are {'micro', 'macro', 'samples', 'weighted'} or None, default='weighted'.
     """
 
     valid_labels: Optional[List[str]] = None
     converter_fn: Callable[[str, List[str]], str] = convert_model_output_to_label
-    multiclass_average_strategy: Optional[str] = "micro"
+    binary_average_strategy: Optional[str] = "binary"
```

Comment:
I very much agree with Pola; "binary" is pretty much the only option that makes sense here. I suppose giving the user freedom to choose other options is fine, since we've configured the default appropriately. If they choose something like "micro" and are confused why precision and recall are always the same, that's a self-inflicted customer error.